rust/processor/migrations/2024-05-22-200847_add_v1_migration_views/up.sql

```diff
@@ -52,11 +50,26 @@ SELECT transaction_version,
     inserted_at
 FROM public.fungible_asset_balances
 WHERE token_standard = 'v1';
+-- replace `coin_infos` with `fungible_asset_metadata`
+CREATE OR REPLACE VIEW legacy_migration_v1.coin_infos AS
+SELECT encode(sha256(asset_type::bytea), 'hex'),
```

Comment:
do we need to rename this?  as `coin_type_hash`

Reply:
Yes! Good catch.